blender/arm/make_renderpath.py

```diff
@@ -301,19 +325,31 @@ def build():
         assets.add_khafile_def('rp_stereo')
         assets.add_khafile_def('arm_vr')
         wrd.world_defs += '_VR'
+        assets.add(assets_path + 'vr.png')
+        assets.add_embedded_data('vr.png')
```

Comment:
Is this related to voxel GI?

Reply:
I have no clue. Anyway this file doesn't exist in current version.